config/application.rb

```diff
@@ -29,7 +29,11 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
-    config.load_defaults 7.0
+    config.load_defaults 7.1
+
+    # Force belongs_to validations even on empty/unset keys.
+    #   This is potentially a Rails bug (?!?) and will be reported to their upstream repository
```

Comment:
Can you put the issue link here so that it can be tracked whether this was in fact a bug / so that any potential fixes can be referenced from within the code?

Reply:
Done :heavy_check_mark: